test-common/pom.xml

```diff
@@ -35,6 +35,17 @@
             <artifactId>assertj-core</artifactId>
             <version>3.16.1</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
```

Comment:
can you add a comment in the commit description and point to release not for future reference ... i'm sure will need it!

Reply:
in fact i quickly tried on my side, i doesn't seem to be needed. Please try without it now.